coderd/database/migrations/000402_workspace_app_statuses_app_id_index.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX workspace_app_statuses_app_id_idx ON workspace_app_statuses (app_id);
```

Comment:
Since we're often looking for the _latest_ status, it would help to make this `(app_id, created_at)`

Reply:
I added the sorting to `GetWorkspaceAppStatusesByAppIDs` as well. Given that the index is already sorted, the overhead should be minimal.